frontend/app/components/api/api-v3/hal-resources/work-package-resource.service.ts

```diff
@@ -122,7 +123,6 @@ export class WorkPackageResource extends HalResource {
 
   public $embedded: WorkPackageResourceEmbedded;
   public $links: WorkPackageResourceLinks;
-  public id: string;
```

Comment:
This leads to an error after assigning a work package to a parent in the relations tab. It tries to set a work package resource's ID which is not possible anymore since while you did define a getter for `id` you did not define a setter.